spring-boot-parent/pom.xml

```diff
@@ -473,7 +473,7 @@
 						<java.security.egd>file:/dev/./urandom</java.security.egd>
 						<java.awt.headless>true</java.awt.headless>
 					</systemPropertyVariables>
-					<argLine>-Xmx1024m -XX:MaxPermSize=256m</argLine>
+					<argLine>-Xmx1024m</argLine>
```

Comment:
This would work for "us" but it might break users using the parent on Java7 runtime. Maybe a profile to conditionally change the parameters?